pkgs/containers/nvidia-ctk.nix

```diff
@@ -0,0 +1,22 @@
+{ fetchFromGitLab
+, buildGo120Module
+}:
+buildGo120Module rec {
```

Comment:
Why go120 specifically? If it's needed specifically, could you add a comment explaining why?

Reply:
Their go module requires at least go 1.20. There are MRs on their repo for relaxing this to 1.17, but that is not merged as of right now. I can apply those patches and stick to the default `buildGoModule` though :thinking:

Reply:
I applied the patches to allow for older versions of go and all works :+1: